src/sagemaker/rl/estimator.py

```diff
@@ -362,10 +362,10 @@ def _validate_framework_format(cls, framework):
         Args:
             framework:
         """
-        if framework and framework not in RLFramework:
+        if framework and framework not in list(RLFramework):
```

Comment:
I did some Googling to see if there's a more Pythonic way of doing this, and the best I found was:

```python
if framework and framework not in RLFramework.__members__
```

I don't have a strong preference, though